pkg/clustertask/clustertask.go

```diff
@@ -117,3 +119,47 @@ func getV1alpha1(c *cli.Clients, clustertaskname string, opts metav1.GetOptions)
 	}
 	return clustertask, nil
 }
+
+func Create(c *cli.Clients, ct *v1beta1.ClusterTask, opts metav1.CreateOptions) (*v1beta1.ClusterTask, error) {
+	gvr, err := actions.GetGroupVersionResource(clustertaskGroupResource, c.Tekton.Discovery())
+	if err != nil {
+		return nil, err
+	}
+
+	if gvr.Version == "v1alpha1" {
```

Comment:
Do you see a case for this, can you give example in which scenario this would be needed